dataplane/vppagent/cmd/vppagent-dataplane.go

```diff
@@ -35,6 +33,8 @@ func main() {
 		syscall.SIGTERM,
 		syscall.SIGQUIT)
 
+	go common.BeginHealthCheck()
```

Comment:
Thanks for fixing this... I'm sensitive to it because I've been bitten by it before ;)